### ISSUE TITLE
View [helper] error when updating IDE helper stubs

### ISSUE THREAD

**Issue by dniccum** [honeybadger-io/honeybadger-laravel#80]
## What are the steps to reproduce this issue?

1. Updated to version 3.8
2. I am using the `barryvdh/laravel-ide-helper` Composer package (v2.9.1), so when the update is complete I am running `php artisan ide-helper:generate` to update my IDE stubs.

## What happens?

I am receiving an error referencing a missing view:

```
 InvalidArgumentException 

  View [helper] not found.

  at vendor/laravel/framework/src/Illuminate/View/FileViewFinder.php:137
    133▕                 }
    134▕             }
    135▕         }
    136▕ 
  ➜ 137▕         throw new InvalidArgumentException("View [{$name}] not found.");
    138▕     }
    139▕ 
    140▕     /**
    141▕      * Get an array of possible view files.

  • helper was not found: Are you sure the view exists and is a `.blade.php` file? 

      +23 vendor frames 
  24  artisan:37
      Illuminate\Foundation\Console\Kernel::handle(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
```

## What were you expecting to happen?

I am expecting this response from the `php artisan ide-helper:generate` command:

```
A new helper file was written to _ide_helper.php
```

## Any logs, error output, etc?

See above.

## Any other comments?

I was able to resolve the issue by rolling back the HoneyBadger package to version 3.7.0

## What versions are you using?
**Operating System:** Mac OS 11.2.3
**Package Version:** 3.8.0

**Comment by shalvah** [comment]
Thanks for the report! I suspect this is due to this line in the view breadcrumb failing:

https://github.com/honeybadger-io/honeybadger-laravel/blob/b1b38254d9848d12b75432a8cc59526d60d95516/src/Breadcrumbs/ViewRendered.php#L26

If you run the command again with `--verbose`, we can see the full stack trace and confirm this. Alternatively, can you try going back to 3.8.0 and disabling the `ViewRendered` breadcrumb and see if the error goes away? You can simply comment it out in the `breadcrumbs.automatic` array of your `honeybadger.php`. (If you don't have this array in you config, you can copy the one in [the docs](https://docs.honeybadger.io/lib/php/guides/breadcrumbs.html#automatic-breadcrumbs).)

**Comment by dniccum** [comment]
All of the above appears to be correct. I was able to resolve it with the recommended modification to the `honeybadger.php` configuration file.

This is what I get when I run with the `--verbose` flag:

```bash
 InvalidArgumentException 

  View [helper] not found.

  at vendor/laravel/framework/src/Illuminate/View/FileViewFinder.php:137
    133▕                 }
    134▕             }
    135▕         }
    136▕ 
  ➜ 137▕         throw new InvalidArgumentException("View [{$name}] not found.");
    138▕     }
    139▕ 
    140▕     /**
    141▕      * Get an array of possible view files.

  • helper was not found: Are you sure the view exists and is a `.blade.php` file? 

  1   vendor/laravel/framework/src/Illuminate/View/FileViewFinder.php:79
      Illuminate\View\FileViewFinder::findInPaths("helper")

  2   vendor/honeybadger-io/honeybadger-laravel/src/Breadcrumbs/ViewRendered.php:26
      Illuminate\View\FileViewFinder::find("helper")

  3   vendor/honeybadger-io/honeybadger-laravel/src/Breadcrumbs/Breadcrumb.php:27
      Honeybadger\HoneybadgerLaravel\Breadcrumbs\ViewRendered::handleEvent("composing: helper")

  4   vendor/laravel/framework/src/Illuminate/Events/Dispatcher.php:389
      Honeybadger\HoneybadgerLaravel\Breadcrumbs\Breadcrumb::handle("composing: helper")

  5   vendor/laravel/framework/src/Illuminate/Events/Dispatcher.php:237
      Illuminate\Events\Dispatcher::Illuminate\Events\{closure}("composing: helper")

  6   vendor/laravel/framework/src/Illuminate/View/Concerns/ManagesEvents.php:177
      Illuminate\Events\Dispatcher::dispatch("composing: helper")

  7   vendor/laravel/framework/src/Illuminate/View/View.php:120
      Illuminate\View\Factory::callComposer(Object(Illuminate\View\View))

  8   vendor/laravel/framework/src/Illuminate/View/View.php:91
      Illuminate\View\View::renderContents()

  9   vendor/barryvdh/laravel-ide-helper/src/Generator.php:82
      Illuminate\View\View::render()

  10  vendor/barryvdh/laravel-ide-helper/src/Console/GeneratorCommand.php:116
      Barryvdh\LaravelIdeHelper\Generator::generate()

  11  vendor/laravel/framework/src/Illuminate/Container/BoundMethod.php:36
      Barryvdh\LaravelIdeHelper\Console\GeneratorCommand::handle()

  12  vendor/laravel/framework/src/Illuminate/Container/Util.php:40
      Illuminate\Container\BoundMethod::Illuminate\Container\{closure}()

  13  vendor/laravel/framework/src/Illuminate/Container/BoundMethod.php:93
      Illuminate\Container\Util::unwrapIfClosure(Object(Closure))

  14  vendor/laravel/framework/src/Illuminate/Container/BoundMethod.php:37
      Illuminate\Container\BoundMethod::callBoundMethod(Object(Illuminate\Foundation\Application), Object(Closure))

  15  vendor/laravel/framework/src/Illuminate/Container/Container.php:611
      Illuminate\Container\BoundMethod::call(Object(Illuminate\Foundation\Application), [])

  16  vendor/laravel/framework/src/Illuminate/Console/Command.php:136
      Illuminate\Container\Container::call()

  17  vendor/symfony/console/Command/Command.php:256
      Illuminate\Console\Command::execute(Object(Symfony\Component\Console\Input\ArgvInput), Object(Illuminate\Console\OutputStyle))

  18  vendor/laravel/framework/src/Illuminate/Console/Command.php:121
      Symfony\Component\Console\Command\Command::run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Illuminate\Console\OutputStyle))

  19  vendor/symfony/console/Application.php:971
      Illuminate\Console\Command::run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))

  20  vendor/symfony/console/Application.php:290
      Symfony\Component\Console\Application::doRunCommand(Object(Barryvdh\LaravelIdeHelper\Console\GeneratorCommand), Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))

  21  vendor/symfony/console/Application.php:166
      Symfony\Component\Console\Application::doRun(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))

  22  vendor/laravel/framework/src/Illuminate/Console/Application.php:92
      Symfony\Component\Console\Application::run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))

  23  vendor/laravel/framework/src/Illuminate/Foundation/Console/Kernel.php:129
      Illuminate\Console\Application::run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))

  24  artisan:37
      Illuminate\Foundation\Console\Kernel::handle(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
```

**Comment by shalvah** [comment]
This should be fixed in 3.8.1. Can you test and confirm? You can uncomment the ViewRendered breadcrumb.

**Comment by dniccum** [comment]
Yes this has been fixed. Thank you.
